net/pfSense-pkg-Avahi/files/usr/local/www/avahi_settings.php

```diff
@@ -158,6 +159,13 @@
 	$pconfig['enable']
 ));
 
+$section->addInput(new Form_Select(
+	'carpstatusvid',
+	'CARP Status IP',
+	$pconfig['carpstatusvid'],
+	avahi_get_carp_list()
+))->setHelp('Used to determine the CARP status. When the CARP vhid is in BACKUP status, Avahi will not be started.');
```

Comment:
Could use better wording:
`Used to determine the HA MASTER/BACKUP status. Avahi will be stopped when the chosen VIP is in BACKUP status, and started in MASTER status.'`

Reply:
done